lib/private/Accounts/AccountManager.php

```diff
@@ -138,35 +107,22 @@ class AccountManager implements IAccountManager {
 	];
 
 	public function __construct(
-		IDBConnection            $connection,
-		IConfig                  $config,
-		EventDispatcherInterface $eventDispatcher,
-		IJobList                 $jobList,
-		LoggerInterface          $logger,
-		IVerificationToken       $verificationToken,
-		IMailer                  $mailer,
-		Defaults                 $defaults,
-		IFactory                 $factory,
-		IURLGenerator            $urlGenerator,
-		ICrypto                  $crypto
+		private IDBConnection $connection,
+		private IConfig $config,
+		private EventDispatcherInterface $eventDispatcher,
+		private IJobList $jobList,
+		private LoggerInterface $logger,
+		private IVerificationToken $verificationToken,
+		private IMailer $mailer,
+		private Defaults $defaults,
+		private IFactory $factory,
```

Comment:
```suggestion
		private IFactory $l10nFactory,
```

rename this for readbility/clarity, factory alone is a bit broad

Reply:
@blizzz Thanks. Other than making it more readable, I searched the codebase and almost all other classes also used this convention. So keeping it consistent is always good.